frontend/src/libs/components/survey-category/survey-category.tsx

```diff
@@ -0,0 +1,46 @@
+import { useCallback } from '#libs/hooks/hooks.js';
+import {} from '#libs/types/form/form.type.js';
```

Comment:
i guess you forgot to remove this 
import {} from '#libs/types/form/form.type.js';

Reply:
Oh really, thanks